products/ansible.md

```diff
@@ -19,9 +19,15 @@ auto:
 -   pypi: ansible
 
 releases:
+-   releaseCycle: "8"
+    releaseDate: 2023-05-30
+    eol: false
+    latest: "8.0.0"
+    latestReleaseDate: 2023-05-30
+
 -   releaseCycle: "7"
     releaseDate: 2022-11-22
-    eol: false
+    eol: 2023-05-30
```

Comment:
this is wrong, ansible@7 would be EOL after 7.7.0

See release cycle doc, https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html#ansible-community-package-release-cycle